src/screens/Home/Start.js

```diff
@@ -41,6 +41,17 @@ Dot.propTypes = {
   color: PropTypes.string.isRequired,
 };
 
+const isDarkMode =false;
+
+if (
+  themeMode === 'auto' &&
```

Comment:
themeMode isn't defined

Reply:
thanks for reviewing it fixed all the changes u suggested

Reply:
@jcfilben please review this pr as this would contribute toward my hacktoberfest contribution

Reply:
@britt6612 could you please review this pr as this would contribute toward my hacktoberfest contribution